src/dotnet-install.ps1

```diff
@@ -635,6 +647,12 @@ function Prepend-Sdk-InstallRoot-To-Path([string]$InstallRoot, [string]$BinFolde
     }
 }
 
+Say "Please note that the intended use of this script is for Continuous Integration (CI) scenarios, where:"
+Say "- The SDK needs to be installed without user interaction and without admin rights."
+Say "- The SDK installation doesn't need to persist across multiple CI runs."
+Say "To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer."
+Say-Text-Separator
```

Comment:
minor: Does System.Environment.NewLine not get the job done instead of needing `Say-Text-Separator`?

Reply:
I understand that the function is used to print a new line without prefixing it with `dotnet-install:`. However, we can achieve the same effect if we appended a new line to the previous message, right? The last line would look like this:

```
Say "To set up a dev... ... to get the installer.`r`n"
```

Can we get rid of `Say-Text-Separator` afterwards?

Reply:
That is a good idea, yes, this will give the needed effect. I'll modify scripts as @bozturkMSFT suggests.